.github/workflows/daily.yaml

```diff
@@ -1,36 +1,35 @@
-name: Daily Test
+name: Daily Integration Test
 
 on:
-  workflow_dispatch: # run on request (no need for PR)
+  workflow_dispatch: # Allows manual trigger
   schedule:
-    # every UTC 7PM from Mon to Fri
-    - cron: "0 19 * * 1-5"
+    # every UTC/GMT 8PM from Mon to Fri
+    - cron: "0 20 * * 1-5"
 
 # Declare default permissions as read only.
 permissions: read-all
 
 jobs:
-  Daily-e2e-tests:
-    runs-on: [otx-gpu-a10g-1]
-    container:
-      image: 219678651685.dkr.ecr.eu-west-1.amazonaws.com/ote-ci:11.7.1.2-devel-ubuntu20.04
-      options: "--runtime=nvidia --env-file=/home/runner/.nvidia.env --ipc=host"
+  Daily-Integration-Tests:
+    runs-on: [otx-ci-3090-03-all]
```

Comment:
Will we use self hosted machine instead of AWS for daily, right?

Reply:
yeah, that's the idea. We wanna reduce the usage on AWS runner